test/JDBC/expected/BABEL-3556-vu-prepare.out

```diff
@@ -0,0 +1,266 @@
+-- To test out of range values for fixeddecimal
+CREATE TABLE babel_3556_vu_prepare_t1(a fixeddecimal);
+GO
+
+CREATE TABLE babel_3556_vu_prepare_t2(a fixeddecimal);
+GO
+
+-- decimal
+SELECT CAST(922337203685478.5807 AS fixeddecimal);
```

Comment:
1. Let's use `view` for every select statements
2. Usually, we don't need to test select output in prepare scripts

For example,
Preparation script 
```
create view babel_3556_vu_prepare_v1 as SELECT CAST(922337203685478.5807 AS fixeddecimal);
```

Verification script
```
select * from babel_3556_vu_prepare_v1
```